penumbra/src/bin/pcli.rs

```diff
@@ -117,10 +117,10 @@ async fn main() -> Result<()> {
         }
         Command::Wallet(Wallet::Generate) => {
             if wallet_path.exists() {
-                Err(anyhow::anyhow!(
+                return Err(anyhow::anyhow!(
                     "Wallet path {} already exists, refusing to overwrite it.",
                     wallet_path.display()
-                ))?;
```

Comment:
This `?` is always going to throw an error, so we can instead unconditionally `return` the `Err`